Marlin/src/module/temperature.cpp

```diff
@@ -86,17 +86,17 @@ Temperature thermalManager;
  */
 
 #if HAS_HEATED_BED
-  #define _BED_PSTR(E) (E) == -1 ? PSTR(MSG ## _BED) :
+  #define _BED_PSTR(M,E) (E) == -1 ? PSTR(MSG_BED " " M) :
```

Comment:
We tend not to do this kind of appending of strings because in some languages the order is different, and we don't want to impose our English-centric word ordering on everyone.